src/Web/WebSPA/Client/modules/campaigns/campaigns.component.ts

```diff
@@ -22,14 +22,14 @@ export class CampaignsComponent implements OnInit {
 
     ngOnInit() {
         if (this.configurationService.isReady) {
+            this.isCampaignDetailFunctionEnabled = this.configurationService.serverSettings.activateCampaignDetailFunction;
```

Comment:
I think this module is related to the marketing functionality, so the whole thing should actually be removed